cli/eventstore/eventstore.spec.js

```diff
@@ -0,0 +1,45 @@
+
+const { 
+    createEventStore, 
+    syncEventStore, 
+    writeEvent 
+} = require('./eventstore')
+
+var ethUtils = require('ethereumjs-util')
+
+contract('Transmute CLI', (accounts) => {
+    let account0 = accounts[0]
```

Comment:
This is a valid variable name, but if we are going to include numbers like this, i would prefer something like `act0`. But that's just me really not liking the way that looks.